react-native-libraries.json

```
@@ -9974,7 +9974,8 @@
       "https://raw.githubusercontent.com/maplibre/maplibre-react-native/main/assets/indoor_building_map_ios.png"
     ],
     "ios": true,
-    "android": true
+    "android": true,
+    "newArchitecture": true
```

```suggestion
    "newArchitecture": true,
    "newArchitectureNote": "Supported from v10 release"
```

Let's add a note about the new support based on your testing.